modules/oc-mirror-signature-verification.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-about-sig-mirroring-verification_{context}"]
+= Enabling signature verification for oc-mirror plugin v2
+
+Starting with {product-title} 4.19, oc-mirror plugin v2 supports signature verification, which is disabled by default. When enabled, the plugin verifies that container images match their signatures, ensuring they have not been altered and come from trusted sources. If a signature mismatch is detected, the mirroring workflow will fail.
+
+.Procedure
+
+. To enable signature verification for all images, run the following command:
```

Comment:
I would rephrase this as an "if" statement, both because it more closely follows ISG guidance for conditional steps, and because it would be parallel with how the other procedure is phrased:
```suggestion
. If you want to enable signature verification for all images, run the following command:
```